Documentation/Communicational Patterns/2. Tooling/2.4. ObservedObject.md

```diff
@@ -0,0 +1,46 @@
+![document type: vision](https://themomax.github.io/resources/markdown-labels/document_type_vision.svg)
+
+# ObservedObject
+
+The `@ObservedObject` wraps an `ObservableObject`. An `ObservableObject` may have multiple `@Published` properties. Each `@Published` property is projected to a `Publisher`. A `Handler` that contains an `@ObservedObject` subscribes to these `Publisher`s and evaluates the `Handler` on each update.
+
+## Lifetime
+
+### Influence
+
+The presence of `@ObservedObject` properties on a `Handler` signalizes exporting the endpoint as a Server-Side Stream makes sense. If the exporter decides to do so the lifetime of the `Handler` has to be extended accordingly. The `Handler` should stay alive until all `Publisher`s projected from observed properties have finished.
+
+An exception is made for `@Published(.weak)`. `Handler`s do not wait for weakly published properties to finish before being destructed.
```

Comment:
Looking at possible forms of reuse I would suggest to keep the `Combine` API the same as it can be found on Apple Platforms or OpenCombine and only offer an `@Published` property wrapper and a `.end(Reponse)` `Action`.

Reply:
I see your point. I agree that having a `.end(Response)` is the superior approach. The initial counter-argument I saw with this approach was that you cannot have a silent endpoint, that never sends a service-message but only consumes requests (e.g. supported by WebSocket). Thus I created two new cases: `.end` and `.final(Response)`.